data/en/test_variants_question.json

```diff
@@ -0,0 +1,482 @@
+                                    "label": "First Name",
+                                    "mandatory": true,
+                                    "type": "TextField"
+                                },
+                                {
+                                    "id": "last-name-answer",
+                                    "description": "",
+                                    "label": "Last Name",
+                                    "mandatory": false,
+                                    "type": "TextField"
+                                }
+                            ]
+                        }
+                    },
+                    {
+                        "type": "Question",
+                        "id": "proxy-block",
+                        "question": {
+                            "id": "proxy-question",
+                            "title": {
+                                "text": "Are you <em>{person_name}</em>?",
+                                "placeholders": [{
+                                    "placeholder": "person_name",
+                                    "transforms": [{
+                                        "transform": "concatenate_list",
+                                        "arguments": {
+                                            "list_to_concatenate": {
+                                                "source": "answers",
+                                                "identifier": ["first-name-answer", "last-name-answer"]
+                                            },
+                                            "delimiter": " "
+                                        }
+                                    }]
+                                }]
+                            },
+                            "type": "General",
+                            "answers": [{
+                                "id": "proxy-answer",
+                                "mandatory": true,
+                                "options": [{
+                                        "label": "Yes, I am",
+                                        "value": "non-proxy"
+                                    },
+                                    {
+                                        "label": "No, I am answering on their behalf",
+                                        "value": "proxy"
+                                    }
+                                ],
+                                "type": "Radio"
+                            }]
+                        }
+                    }
+                ]
+            }]
+        },
+        {
+            "id": "basic-question-variant-section",
+            "groups": [{
+                "id": "basic-question-variant-group",
+                "title": "Variants",
+                "blocks": [{
+                        "type": "Question",
+                        "id": "age-block",
+                        "title": "questions only",
```

Comment:
The block titles should be removed. We don't use them in real questionnaires and they aren't adding anything.